packages/client/helpers/build.ts

```diff
@@ -89,6 +84,11 @@ function bundleTypeDefinitions(filename: string, outfile: string) {
       ],
       compiler: {
         tsconfigFilePath: 'tsconfig.build.json',
+        overrideTsconfig: {
+          compilerOptions: {
+            paths: {}, // bug with api extract + paths
```

Comment:
So this is a workaround? What's the bug? (is there an issue to link?)

Reply:
I might need to report it, basically, when paths are in use, it bundles entire classes an not just the class definitions.

Reply:
That leads to an invalid `d.ts` bundle.